packages/lib/src/vis/shared/PanZoomMesh.tsx

```diff
@@ -29,10 +30,14 @@ function PanZoomMesh() {
         abscissaScale.invert(x),
         ordinateScale.invert(y)
       );
+      const { position, projectionMatrixInverse } = camera;
+
+      const cameraBounds = CAMERA_TOP_RIGHT.clone().applyMatrix4(
```

Comment:
The projection is this time done from camera (normalized) space to local camera space. 

The reason is that the local camera space is centered on `camera.position`. That means the projection of the top right point is enough to get the bounds of the camera FOV in X and Y. Projecting the bottom left point would get the same values with a `-`.

Reply:
`cameraLocalBounds` and maybe a comment explaining this part "Projecting the bottom left point would get the same values with a -."?